packages/fx-core/tests/core/hooks.test.ts

```diff
@@ -632,6 +632,7 @@ describe("Middleware - others", () => {
       await fs.ensureDir(projectPath);
       mockedEnvRestore = mockedEnv({
         TEAMSFX_INSIDER_PREVIEW: "true",
+        AAD_APP_CLIENT_SECRET: "test",
```

Comment:
do we need to set this environment variable?

Reply:
fixed, add ignoreEnvInfo to inputs as the common load logic